base_import_match/models/base_import.py

```diff
@@ -42,9 +42,12 @@ def _onchange_model_id(self):
     def _compute_name(self):
         """Automatic self-descriptive name for the setting records."""
         for one in self:
+            display_names = one.field_ids.filtered(lambda f: f.display_name).mapped(
```

Comment:
`filtered(lambda f: f.display_name)` is equivalent to just `filtered("display_name")`

Reply:
@dreispt Thank you, change applied